static/app/views/organizationIntegrations/integrationRow.tsx

```diff
@@ -33,6 +33,7 @@ type Props = {
   configurations: number;
   categories: string[];
   alertText?: string;
+  resolveText?: string;
```

Comment:
can you add a comment here about what this is / when to use it for future users of this?